app/components/transUnit/TransUnitService.js

```diff
@@ -190,7 +237,9 @@
     }
 
     function setSelected(transUnitCtrl, isSelected) {
-      transUnitCtrl.selected = isSelected || false;
+      if(transUnitCtrl.selected !== isSelected) {
+        transUnitCtrl.selected = isSelected || false;
+      }
```

Comment:
@aeng never mind, I was still looking at the outdated diff so I didn't see that you already added a comment.